modules/update-oc-configmap-signature-verification.adoc

```diff
@@ -11,6 +11,7 @@ Before you update your cluster, you must manually create a config map that conta
 ====
 If you are upgrading from a release prior to version 4.4.8, you must use the manual method for creating the config map instead of this procedure. The commands that this procedure uses are not in earlier versions of the `oc` command-line interface (CLI).
 ====
+//Can we remove this note?
```

Comment:
@LalatenduMohanty  Here is a second - Would you review this for  OpenShift 4.9 GA?

Reply:
I'm in favor of removing this note, originally from [here][1].  Other old 4.4.8 refs:

```console
$ git --no-pager grep '4[.]4[.]8' origin/main
origin/main:modules/update-oc-configmap-signature-verification.adoc:If you are upgrading from a release prior to version 4.4.8, you must use the manual method for creating the config map instead of this procedure. The commands that this procedure uses are not in earlier versions of the `oc` command-line interface (CLI).
origin/main:modules/update-oc-configmap-signature-verification.adoc:* Install the OpenShift CLI (`oc`), version 4.4.8 or later.
origin/main:modules/update-restricted.adoc:* Install the OpenShift CLI (`oc`), version 4.4.8 or later.
origin/main:updating/updating-restricted-network-cluster.adoc:If you are upgrading from version 4.4.8 or later, you can use the `oc` CLI to create the config map. If you are upgrading from an earlier version, you must use the manual method.
```

[1]: https://github.com/openshift/openshift-docs/pull/21993/files#diff-e41f68fc4d62522b5de6fb2ae40dea4beb75eeebc1d3e8d4a8b60bde8a376cf7R12

Reply:
yes, we can remove it. I am assuming that we only removing this from the 4.9 docs but it will continue to exist in previous docs. Which is the right thing to do.

Reply:
@LalatenduMohanty, yes, this would only be removed from 4.9+. Thanks for confirming!